.gitignore

```diff
@@ -39,4 +39,9 @@ yarn-error.log
 NTH
 
 docs
-package-lock.json
\ No newline at end of file
+package-lock.json
+yarn.lock
```

Comment:
lock files should be commited at some point... @fed135 we will talk about it later?

Reply:
Taken from other open-source projects, committing lockfiles adds restrictions to the range of dependencies that consumers can have in their applications. This helps to reduce the overall amount of dependencies in projects.